tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_validate_xml_feed_content.yaml

```diff
@@ -1,208 +1,198 @@
   metadata:
-    provider_name: 'Ubuntu Xenial'
-    expected_format: 'application/x-bzip2'
-    path: '/tmp/com.ubuntu.xenial.cve.oval.xml.bz2'
-    extension: 'bz2'
-    decompressed_file: '/tmp/xenial.xml'
-    url: 'https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.xenial.cve.oval.xml.bz2'
+    provider_name: Ubuntu Xenial
+    expected_format: application/x-bzip
+    path: /tmp/com.ubuntu.xenial.cve.oval.xml.bz2
+    extension: bz2
+    decompressed_file: /tmp/xenial.xml
+    url: https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.xenial.cve.oval.xml.bz2
 
-- name: 'Canonical Trusty'
-  description: 'Canonical provider'
+- name: Canonical Trusty
+  description: Canonical provider
   configuration_parameters:
   metadata:
-    provider_name: 'Ubuntu Trusty'
-    expected_format: 'application/x-bzip2'
-    path: '/tmp/com.ubuntu.trusty.cve.oval.xml.bz2'
-    extension: 'bz2'
-    decompressed_file: '/tmp/trusty.xml'
-    url: 'https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.trusty.cve.oval.xml.bz2'
+    provider_name: Ubuntu Trusty
+    expected_format: application/x-bzip
+    path: /tmp/com.ubuntu.trusty.cve.oval.xml.bz2
+    extension: bz2
+    decompressed_file: /tmp/trusty.xml
+    url: https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.trusty.cve.oval.xml.bz2
 
-- name: 'Debian'
-  description: 'Debian provider'
+- name: Debian
+  description: Debian provider
   configuration_parameters:
   metadata:
-    provider_name: 'Debian Buster'
-    expected_format: 'xml'
-    path: '/tmp/oval-definitions-buster.xml'
-    extension: 'xml'
-    url: 'https://www.debian.org/security/oval/oval-definitions-buster.xml'
+    provider_name: Debian Buster
+    expected_format: xml
+    path: /tmp/oval-definitions-buster.xml
+    extension: xml
+    url: https://www.debian.org/security/oval/oval-definitions-buster.xml
 
-- name: 'Debian'
-  description: 'Debian provider'
+- name: Debian
+  description: Debian provider
   configuration_parameters:
   metadata:
-    provider_name: 'Debian Stretch'
-    expected_format: 'xml'
-    path: '/tmp/oval-definitions-stretch.xml'
-    extension: 'xml'
-    url: 'https://www.debian.org/security/oval/oval-definitions-stretch.xml'
+    provider_name: Debian Jessie
+    expected_format: xml
+    path: /tmp/oval-definitions-jessie.xml
+    extension: xml
+    url: https://www.debian.org/security/oval/oval-definitions-jessie.xml
 
-- name: 'Debian'
-  description: 'Debian provider'
+- name: Debian
+  description: Debian provider
   configuration_parameters:
   metadata:
-    provider_name: 'Debian Jessie'
-    expected_format: 'xml'
-    path: '/tmp/oval-definitions-jessie.xml'
-    extension: 'xml'
-    url: 'https://www.debian.org/security/oval/oval-definitions-jessie.xml'
+    provider_name: Debian Wheezy
+    expected_format: xml
+    path: /tmp/oval-definitions-wheezy.xml
+    extension: xml
+    url: https://www.debian.org/security/oval/oval-definitions-wheezy.xml
```

Comment:
Wheezy and Jessie are deprecated, please remove them from all tests

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3172/commits/dafc1ca26541ede97f9cf75e9f12a5fdc56796f0